HelpSource/Classes/SoundFileAnalysis.schelp

```diff
@@ -142,6 +144,21 @@ ARGUMENT::maxDataPoints
 
 A limit on the number of points added to the data.
 
+ARGUMENT::trimDuration
+
+Until now the default behaviour SoundFileAnalysis is that the duration is trimmed at least to the length of a file. Additionally because only full blocks are written to the buffer usually the last samples of a file are not analysed. because of this we introduced trimDuration.
```

Comment:
In the helpfile, better describe it as if it had been like this forever:

```
When code::trimDuration:: is unspecified, the duration is no longer than the length of the file. Because only full blocks are written to the buffer, usually the last samples of a file are not analysed. Setting code::trimDuration:: allows you to specify what is supposed to happen with the last block.